components/match2/commons/starcraft_starcraft2/opponent_display_starcraft.lua

```diff
@@ -241,6 +241,28 @@ end
 
 StarcraftOpponentDisplay.CheckMark = '<i class="fa fa-check forest-green-text" aria-hidden="true"></i>'
 
+StarcraftOpponentDisplay.propTypes.BlockScore = {
+	isWinner = 'boolean?',
+	scoreText = 'any',
+	side = 'string',
+}
+
+--[[
+Displays a score within the context of a block element.
+]]
+function StarcraftOpponentDisplay.BlockScore(props)
+	DisplayUtil.assertPropTypes(props, StarcraftOpponentDisplay.propTypes.BlockScore)
+
+	local scoreText = props.scoreText
+	if props.isWinner then
+		scoreText = '<b>' .. scoreText .. '</b>'
+	end
+
+	return html.create('div')
+		:wikitext(scoreText)
+		:addClass('brkts-popup-header-opponent-score-' .. (props.side or 'left'))
```

Comment:
The css class name should match the module or component name. Maybe `starcraft-block-score`? Or `block-score` if made not sc specific.

If using flip=true/false then this would be 
```
:addClass('starcraft-block-score')
:addClass(props.flip and 'flipped' or nil)
```

Reply:
moved the class adding to matchsummary